docs/details/dashboards/dashboard-replsetsummary-new.md

```diff
@@ -0,0 +1,165 @@
+
+# MongoDB ReplSet Summary (NEW)
```

Comment:
Consider to remove "(NEW)". It is not included in the dashboard name (in the product)